src/main/java/com/epam/spring/boot/cargodeliverysystem/controller/CityController.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.spring.boot.cargodeliverysystem.controller;
+
+import com.epam.spring.boot.cargodeliverysystem.dto.CityDto;
+import com.epam.spring.boot.cargodeliverysystem.service.CityService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequestMapping("/cities")
```

Comment:
Total nitpick, but I would use singular not plural, like /city. The same in other controllers

Reply:
fixed